### PR TITLE
Update edition-83.md

### PR DESCRIPTION
**Pull request by PhilipOakley**
Add three light reading articles I'd noted in the last month.

Signed-off-by: Philip Oakley <philipoakley@iee.email>

### CODE REVIEW

**Review by mjaix — APPROVED** [review]

File: rev_news/drafts/edition-83.md [hunk 1]
```diff
@@ -171,6 +171,12 @@ __Light reading__
   by Michael Meli, Matthew R. McNiece, Bradley Reaves in NDSS 2019;
   via [the morning paper article](https://blog.acolyer.org/2019/04/08/how-bad-can-it-git-characterizing-secret-leakage-in-public-github-repositories/)
   by Adrian Colyer.
+* [The Pull Request Paradox: Merge Faster by Promoting Your PR](https://dzone.com/articles/the-pull-request-paradox-merge-faster-by-promoting)
+  Adding more context to our Pull Requests got them merged two days faster. Here's how it works. By Dan Lines.
+* [What's in a Good Error Message?](https://www.morling.dev/blog/whats-in-a-good-error-message/)
+  Some thoughts on ensuring error messages are useful to the reader. By Gunnar Morling.
+* [How to Write Better Git Commit Messages – A Step-By-Step Guide](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/)
+  Ever wondered how you can improve your Git commit messages? Suggestions by Natalie Pina.
```
Comment (chriscool) [inline rev_news/drafts/edition-83.md:179]: This last article had already been added by @jnareb in 9cf8c9fb on line 164. I will remove it in a separate commit.

Comment (chriscool) [reply rev_news/drafts/edition-83.md:179]: Removed the last link in 4ddc884ce9f1fe8dcf9c7bf9db598cc5bd807a5a

Comment (PhilipOakley) [reply rev_news/drafts/edition-83.md:179]: I had a feeling that I'd seen one of them, but I didn't find it when I looked things over. My fault.

Philip

Comment (chriscool) [reply rev_news/drafts/edition-83.md:179]: No worries!

**Pull request merged**